assets/components/kube-apiserver/defaultconfig.yaml

```diff
@@ -167,6 +167,8 @@ apiServerArguments:
     - /etc/kubernetes/static-pod-certs/secrets/service-network-serving-certkey/tls.crt
   tls-private-key-file:
     - /etc/kubernetes/static-pod-certs/secrets/service-network-serving-certkey/tls.key
+  send-retry-after-while-not-ready-once:
```

Comment:
For the moment, MicroShift-specific overrides live in https://github.com/openshift/microshift/blob/2d490cbc3a835eb4c855d08419898ba2a1e89b92/pkg/controllers/kube-apiserver.go#L111 to make it easier to pull in updated base configs from cluster-kube-apiserver-operator.

Reply:
Yes, I saw that, I moved it to here for now, until you have the configuration ironed out.